emr_launcher.py

```diff
@@ -48,9 +52,22 @@ def read_local_config(config_file: str, required: bool = True) -> dict:
             raise
     except:
         raise
-
+    config = replace_values(config, secrets)
     return config
 
+def fetch_secrets(secret_name: str, region_name: str) -> dict:
+    session = boto3.session.Session()
+    client = session.client(service_name="secretsmanager", region_name=region_name)
+    response = client.get_secret_value(SecretId=secret_name)
+    return ast.literal_eval(response["SecretString"])
+
+def replace_values(config_file: dict, secrets: dict) -> dict:
```

Comment:
You correctly pass a `dict` holding key:value pairs into this function from `read_local_config()`. And that dict was built by calling `yaml.safe_load()`. So, there's no need to load the YAML again...instead what I think you need to do is:

1. Change the `config_file` parameter to `config` as it's not a file object or filename you're passing around
2. Loop around all of the key:value pairs of the `config` dict, updating the values with ones that have been secrets-replaced.

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/07b81c0f1886a90187413a30683a8264b937f409